canonical/vanilla-framework
# Darker rule component

## Done

- darkens rule colours
- adds dark example for rules
- updates dark theme background colours

Fixes [WD-4564](https://warthogs.atlassian.net/browse/WD-4564)
Fixes [WD-4683](https://warthogs.atlassian.net/browse/WD-4683)
Fixes [WD-4684](https://warthogs.atlassian.net/browse/WD-4684)

## QA

- Open [demo](https://vanilla-framework-4801.demos.haus/docs/patterns/rule)
- Check updated rule colours
- Review updated documentation:
  - https://vanilla-framework-4801.demos.haus/docs/patterns/rule
  - Light theme rules: https://vanilla-framework-4801.demos.haus/docs/examples/patterns/rule/highlight
  - Muted: https://vanilla-framework-4801.demos.haus/docs/examples/patterns/rule/muted
  - Dark theme rules: https://vanilla-framework-4801.demos.haus/docs/examples/patterns/rule/dark

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical/vanilla-framework/wiki/Release-process#pre-release-tasks)



[WD-4564]: https://warthogs.atlassian.net/browse/WD-4564?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[WD-4683]: https://warthogs.atlassian.net/browse/WD-4683?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[WD-4684]: https://warthogs.atlassian.net/browse/WD-4684?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
